katalog/curator-s3/action.yml

```diff
@@ -26,36 +26,136 @@ actions:
       continue_if_exception: False
       disable_action: False
     filters:
-    - filtertype: pattern
-      kind: prefix
-      value: logstash-
-      exclude:
-    - filtertype: age
-      source: creation_date
-      direction: older
-      unit: days
-      unit_count: ${UNIT_COUNT:1}
-      exclude:
+      - filtertype: pattern
+        kind: regex
+        value: "^(system-|kubernetes-|ingress-controller-|audit-).*$"
+        exclude:
+      - filtertype: age
+        source: name
+        direction: older
+        timestring: "%Y.%m.%d"
+        unit: days
+        unit_count: ${BACKUP_UNIT_COUNT:1}
+        exclude:
   2:
     action: delete_indices
     description: >-
-       Delete indices older than ${UNIT_COUNT:1} days (based on index name), for logstash-
-       prefixed indices. Ignore the error if the filter does not result in an
-       actionable list of indices (ignore_empty_list) and exit cleanly.
+      Delete indices older than ${DELETE_UNIT_COUNT:1} days (based on index name), for logstash-
+      prefixed indices. Ignore the error if the filter does not result in an
+      actionable list of indices (ignore_empty_list) and exit cleanly.
     options:
       ignore_empty_list: True
       timeout_override:
       continue_if_exception: True
       disable_action: False
     filters:
-    - filtertype: pattern
-      kind: prefix
-      value: logstash-
-      exclude:
-    - filtertype: age
-      source: name
-      direction: older
-      timestring: '%Y.%m.%d'
-      unit: days
-      unit_count: ${UNIT_COUNT:1}
-      exclude:
+      - filtertype: pattern
+        kind: regex
+        value: "^(system-|kubernetes-|ingress-controller-|audit-).*$"
+        exclude:
+      - filtertype: age
+        source: name
+        direction: older
+        timestring: "%Y.%m.%d"
+        unit: days
+        unit_count: ${DELETE_UNIT_COUNT:1}
+        exclude:
+
+  3:
+    action: delete_snapshots
+    description: >-
+      Delete old snapshots (older than ${S3_BACKUP_UNIT_RETENTION} days) from repository ('${SNAPSHOT_S3_TASK}' by default).
+    options:
+      repository: ${SNAPSHOT_S3_TASK:s3-backup-override-me}
+      ignore_empty_list: True
+      retry_interval: 120
+      retry_count: 3
+      disable_action: False
+    filters:
+      - filtertype: pattern
+        kind: regex
+        value: "^(system-).*$"
+      - filtertype: age
+        source: name
+        direction: older
+        timestring: "%Y.%m.%d"
+        unit: days
+        unit_count: ${S3_BACKUP_UNIT_RETENTION}
+#   4:
```

Comment:
The commented code is intended?

Reply:
yes @angelbarrera92 , this because of the setup is belonging to tracing (jaeger with elasticsearch backend) that is not something default as the others. So i left them in commented as a possible quick opt in whenever will be useful 👍

Reply:
If I may suggest something, I would like to have it as an example.